processing/src/test/java/org/apache/druid/query/aggregation/first/StringFirstAggregationTest.java

```diff
@@ -115,7 +117,7 @@ public void testCombine()
   {
     SerializablePairLongString pair1 = new SerializablePairLongString(1467225000L, "AAAA");
     SerializablePairLongString pair2 = new SerializablePairLongString(1467240000L, "BBBB");
-    Assert.assertEquals(pair2, stringFirstAggFactory.combine(pair1, pair2));
+    Assert.assertEquals(pair1, stringFirstAggFactory.combine(pair1, pair2));
```

Comment:
This is the fix to the existing test for `StringFirstAggregatorFactory.combine()` fixes.